Lib/idlelib/config_key.py

```diff
@@ -219,33 +224,34 @@ def TranslateKey(self, key, modifiers):
         return key
 
     def OK(self, event=None):
-        if self.advanced or self.KeysOK():  # doesn't check advanced string yet
-            self.result=self.keyString.get()
+        key = self.keyString.get()
+        key.strip()
+        if not key:
+            tkMessageBox.showerror(title=self.keyerror_title, parent=self,
+                                   message="No keys specified.")
```

Comment:
I wasn't sure when you would want the tkMessageBox and tkFont to be updated when changes are made.  I can make a PR for just that change across all source or change it as we come across it, like this.  I saw it on the roadmap and just wasn't sure how you wanted to handle it.